packages/ui/src/styles/ag-theme-vuestic/_ag-theme-vuestic-default-params.scss

```diff
@@ -0,0 +1,28 @@
+@import "~ag-grid-community/dist/styles/ag-theme-base/sass/ag-theme-base-default-params";
+
+$ag-theme-vuestic-default-params: (
```

Comment:
There is stylelint error. Can resolve it? I suggest removing turn off this rule because it works incorrectly with SCSS maps and it is not important for as.